modules/Bio/EnsEMBL/Variation/Pipeline/ReleaseDataDumps/ReleaseDumps_conf.pm

```diff
@@ -70,14 +70,16 @@ sub default_options {
         
         ensembl_registry   => $self->o('ensembl_registry'),
 
-        script_dir         => $self->o('ensembl_cvs_root_dir') . '/ensembl-variation/scripts',
+        hive_root_dir        => '/hps/software/users/ensembl/repositories/' . $ENV{'USER'} . '/ensembl-hive',
```

Comment:
Could $self->o('ensembl_cvs_root_dir') be used instead of specifying path as used for so_file and gvf_readme?

Reply:
Done. Thank you